subprojects/docs/src/docs/release/notes.md

```diff
@@ -221,6 +221,25 @@ See the [toolchain documentation](userguide/toolchains.html) for more in-depth i
 When using [dependency injection](userguide/custom_gradle_types.html#service_injection) when developing plugins, tasks or project extensions, it is now possible to use the `@Inject` annotation without explicitly importing it into your build scripts the
 same way it works for other Gradle API classes.
 
+### Test re-run JUnit XML reporting enhancements
+
+A new opt-in [`mergeReruns` option](javadoc/org/gradle/api/tasks/testing/JUnitXmlReport.html#getMergeReruns--) has been added that changes the “JUnit XML” produced by test tasks.
```

Comment:
I'd add a brief sentence first about what we are referring to that links to https://docs.gradle.org/current/userguide/java_testing.html#test_reporting. It can say something that when running tests with Gradle, an XML report is generated by default that is used by tools such as CI servers.

Reply:
The next part of the context is that test retries are currently not captured by some of the tools.

Reply:
PTAL